core/db/queries.py

```diff
@@ -218,6 +226,18 @@ def outcome_data_query(base_query: Query) -> Query:
         else_=ents.Organisation.organisation_name,
     )
 
+    if join_outcome_info:
```

Comment:
This code is a duplicate of that in `query_extend_with_outcome_filter` minus a filter. But that func if not passed an outcome_categories param will default to None not apply the filter, so we can probably just re-use it here.